src/components/HomeHeader.js

```diff
@@ -61,6 +61,11 @@ class HomeHeader extends React.Component {
               <h3 className="page-title">Litmus</h3>
             </div>
           </a>
+
+          <a href="https://hub.litmuschaos.io/" rel="noopener noreferrer" className="home">
```

Comment:
use react Link tag to add tag.
and don't hardcode the link

Reply:
Sorry about that, never used React before.